src/console.cpp

```diff
@@ -218,7 +218,7 @@ static std::string RemoveUnderscores(std::string name)
 /* static */ void IConsole::AliasRegister(const std::string &name, const std::string &cmd)
 {
 	auto result = IConsole::Aliases().try_emplace(RemoveUnderscores(name), name, cmd);
-	if (!result.second) IConsoleError("an alias with this name already exists; insertion aborted");
+	if (!result.second) IConsolePrint(CC_ERROR, "An alias with the name '%s' already exists; insertion aborted", name.c_str());
```

Comment:
`insertion aborted` is a weird way of saying this. Maybe just remove that part? Doesn't really add value, I guess. (given it is already an error and all)